backend/src/services/wallet_service.py

```
@@ -121,8 +145,30 @@ async def get_wallet_balance(self) -> Dict:
+
+            if not response.value:
+                return 0.0
+
+            total_balance = 0.0
+            for account_info in response.value:
+                 data = account_info.account.data
+                 amount_raw = int.from_bytes(data[64:72], "little")
+                 total_balance += amount_raw
+
+            return float(total_balance)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)